src/tenant/views.py

```diff
@@ -47,6 +48,30 @@ def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
 
+def staff_member_required(f):
```

Comment:
This shouldn't be in the tenant app, because it's not related to Tenant stuff. SHould be more generic since it can be used by any app, which means either utilities, or the project app (hackerspace_online).

Is there a utilities.decorators.py? or hackerspace_online.decorators.py?  Find a good place for it.

Reply:
i put it in `` hackerspace_online.decorators.py``